package.json

```diff
@@ -2,34 +2,64 @@
 	"name": "lisk-js",
 	"version": "0.4.5",
 	"description": "JavaScript library for sending Lisk transactions from the client or server",
-	"main": "./dist-node/index.js",
+	"author": "Lisk Foundation <admin@lisk.io>, lightcurve GmbH <admin@lightcurve.io>",
+	"license": "GPL-3.0",
+	"keywords": [
+		"lisk",
+		"blockchain"
+	],
+	"homepage": "https://github.com/LiskHQ/lisk-js#readme",
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/LiskHQ/lisk-js.git"
+	},
+	"bugs": {
+		"url": "https://github.com/LiskHQ/lisk-js/issues"
+	},
+	"engines": {
+		"node": "=6.12.3",
```

Comment:
I have a feeling this would break yarn and maybe npm for any user trying to install this on a newer version of node or npm. Lisk-js should not be required to run on this exact version of node. `>=6` I think is more appropriate

Reply:
Currently `lisk` core only supports particular version of node.
That is why it is freeze at `6.12.3`

Reply:
But this is not lisk core... the js sdk should be able to run on any version of node or the browser that supports the necessary syntax

Reply:
@vitaly-t

Reply:
Breaks on yarn `v1.3.2` but works fine on npm `v3.10.10` (which ships with Node.js `v6.12.3`). @Tosch110 thoughts?

Reply:
Node LTS is v8. This is just incorrect usage of the engines dict, needs to be fixed. No sdk like this would be limited to such a specific version of node AND package manager. His reasoning was that Lisk core uses v6 but I have a swift sdk for Lisk (matches this api) so I’m only supposed to allow people to use my swift sdk if they are on node v6? It just makes no sense, no one will be able to install this with these requirements. Heck electron is on a much later version of node, nano will probably fail to install this as a dependency

Reply:
Yes I understand the point. However, due to the fact that, there was change in Buffer behaviour, 
lisk-js also supports only v6 now.

I created the follow up issue #561 for this.

Reply:
@AndrewBarba I'm afraid I don't really understand your points about a swift SDK, maybe there's some context I'm missing? Regarding the current proposal for these fields...

From the npm docs:
> You can specify the version of node that your stuff works on:
{ "engines" : { "node" : ">=0.10.3 <0.12" } }

We aren't currently testing multiple versions of node with LiskJS, which is why we're planning to put only one version of node in there. I'm definitely open to the idea that we should allow all node 6 versions. However, there's at least one breaking change in node 8 which breaks LiskJS functionality. So according to the npm documentation—as far as I can see—we definitely shouldn't put a range in engines which allowed the use of node 8.

Do you have any resources which advocate a different way of using the engines field?

yarn is not our recommended package manager, but it does offer the `--ignore-engines` option if you want to ignore our recommendation as well as the fact that we only test LiskJS with one version of node and that's the only version we currently support.

Reply:
The original answer to my concern was that lisk core uses Node v6. My point with the swift sdk was that an SDK using the core API to spec has nothing to do with the software versions running on core. Thats the whole point of an api, i call according to spec and I dont need to know what the core machine is using. But now the answer has changed, that apparently lisk js breaks on Node v8 because of a buffer issue. That is completely different (and valid) reason to use the engines options, but the original reason was incorrect usage of the engines options. I currently use lisk-js 0.5.1 on Node v8 extensively, and also on AWS lambda which uses Node 6.10. I bet a lot of others do this as well. So if tests legitimately do not pass on Node v8, then fine, limit to a range of Node 6. But if you leave it as is, very few people are going to be able to install and use this as they currently are

Reply:
Additionally, this is the main dependency in Lisk Nano which is built on Electron, which uses Node v8. So how could you not support Node v8?